src/Abp/Application/Navigation/MenuItemDefinition.cs

```diff
@@ -37,10 +38,10 @@ public class MenuItemDefinition : IHasMenuItemDefinitions
         public string Url { get; set; }
 
         /// <summary>
-        /// A permission name. Only users that has this permission can see this menu item.
+        /// A permission dependency. Only users that can statify this permission dependency can see this menu item.
         /// Optional.
         /// </summary>
-        public string RequiredPermissionName { get; set; }
```

Comment:
RequiredPermissionName should not be removed since it will be a big breaking change.
Instead, we can mark it Obsolete. We may internally convert requiredPermissionName to PermissionDependency.